nautobot/core/templates/admin/base.html

```diff
@@ -8,55 +8,9 @@
-          onerror="window.location='{% url 'media_failure' %}?filename=select2-bootstrap-0.1.0-beta.10/select2-bootstrap.min.css'">
-    <link rel="stylesheet"
-          href="{% static 'flatpickr-4.6.9/themes/light.min.css' %}"
-          onerror="window.location='{% url 'media_failure' %}?filename=flatpickr-4.6.9/themes/light.min.css'">
-    <link rel="stylesheet" id="dark-theme"
-          href="{% versioned_static 'css/dark.css' %}"
-          onerror="window.location='{% url 'media_failure' %}?filename=css/dark.css'"{% if request.COOKIES|get_item:"theme" != 'dark' %} disabled="disabled"{% endif %} />
-    <link rel="stylesheet" id="base-theme"
-          href="{% versioned_static 'css/base.css' %}"
-          onerror="window.location='{% url 'media_failure' %}?filename=css/base.css'">
-    <link rel="apple-touch-icon" sizes="180x180" href="{%  custom_branding_or_static 'icon_180' 'img/nautobot_icon_180x180.png' %}">
-    <link rel="icon" type="image/png" sizes="32x32" href="{% custom_branding_or_static 'icon_32' 'img/nautobot_icon_32x32.png' %}">
-    <link rel="icon" type="image/png" sizes="16x16" href="{% custom_branding_or_static 'icon_16' 'img/nautobot_icon_16x16.png' %}">
-    <link rel="icon" type="image/png" href="{% custom_branding_or_static 'icon_192' 'img/nautobot_icon_192x192.png' %}" sizes="192x192">
-    <link rel="mask-icon" href="{% custom_branding_or_static 'icon_mask' 'img/nautobot_icon_monochrome.svg' %}" color="#0097ff">
-    <link rel="shortcut icon" href="{% custom_branding_or_static 'favicon' 'img/favicon.ico' %}">
-
-    <script src="{% static 'jquery/jquery-3.6.0.min.js' %}"
-        onerror="window.location='{% url 'media_failure' %}?filename=jquery/jquery-3.6.0.min.js'"></script>
-    <script src="{% static 'jquery-ui-1.13.1/jquery-ui.min.js' %}"
-            onerror="window.location='{% url 'media_failure' %}?filename=jquery-ui-1.13.1/jquery-ui.min.js'"></script>
-    <script src="{% static 'bootstrap-3.4.1-dist/js/bootstrap.min.js' %}"
-            onerror="window.location='{% url 'media_failure' %}?filename=bootstrap-3.4.1-dist/js/bootstrap.min.js'"></script>
-    <script src="{% static 'select2-4.0.13/select2.min.js' %}"
-            onerror="window.location='{% url 'media_failure' %}?filename=select2-4.0.13/select2.min.js'"></script>
-    <script src="{% static 'flatpickr-4.6.9/flatpickr.min.js' %}"
-            onerror="window.location='{% url 'media_failure' %}?filename=flatpickr-4.6.9/flatpickr.min.js'"></script>
-    <script src="{% static 'js/forms.js' %}"
-            onerror="window.location='{% url 'media_failure' %}?filename=js/forms.js'"></script>
-
-    <meta name="msapplication-TileColor" content="#2d89ef">
-    <meta name="theme-color" content="#ffffff">
-    <meta charset="UTF-8">
-    <meta name="viewport" content="initial-scale=1, maximum-scale=1, user-scalable=no, width=device-width">
-
+    {% if not is_popup %}
+        {% include 'inc/media.html' %}
+    {% endif %}
```

Comment:
Don't we need at least some of these includes for the popup case? I'll pull down this branch when I get a chance and see what specifically is needed for the popup case and fix it.